testing/workflows/components/kfctl_go_test.jsonnet

```diff
@@ -0,0 +1,372 @@
+// E2E test for the new go based version of kfctl.
+local params = std.extVar("__ksonnet/params").components.kfctl_go_test;
+
+local k = import "k.libsonnet";
+local util = import "workflows.libsonnet";
+local newUtil = import "util.libsonnet";
+
+// TODO(jlewi): Can we get namespace from the environment rather than
+// params?
+local namespace = params.namespace;
+
+local name = params.name;
+
+local prowEnv = util.parseEnv(params.prow_env);
+local bucket = params.bucket;
+
+// The name for the workspace to run the steps in
+local stepsNamespace = "kubeflow";
+// mountPath is the directory where the volume to store the test data
+// should be mounted.
+local mountPath = "/mnt/" + "test-data-volume";
+// testDir is the root directory for all data for a particular test run.
+local testDir = mountPath + "/" + name;
+// outputDir is the directory to sync to GCS to contain the output for this job.
+local outputDir = testDir + "/output";
+local artifactsDir = outputDir + "/artifacts";
+// Source directory where all repos should be checked out
+local srcRootDir = testDir + "/src";
+// The directory containing the kubeflow/kubeflow repo
+local srcDir = srcRootDir + "/kubeflow/kubeflow";
+
+local runPath = srcDir + "/testing/workflows/run.sh";
+local kfCtlPath = srcDir + "/scripts/kfctl.sh";
```

Comment:
Add TODO to remove script dependency once go CLI support delete?

Reply:
Did it now.